CMakeLists.txt

```diff
@@ -133,8 +133,11 @@ if(MSVC)
       string(REPLACE "/RTC1" "" CMAKE_CXX_FLAGS_DEBUG  "${CMAKE_CXX_FLAGS_DEBUG}")
       string(REPLACE "/RTC1" "" CMAKE_C_FLAGS_DEBUG  "${CMAKE_C_FLAGS_DEBUG}")
     else()
-      # Squeeze binary in Release builds
-      add_link_options(/OPT:REF /OPT:ICF)
+      # Reduce the size of the binary in Release & RelWithDebInfo builds
```

Comment:
```suggestion
      # Reduce the size of the binary in Release & RelWithDebInfo builds
      # We do not set /OPT:ICF here because it has zero effect see: https://github.com/mixxxdj/mixxx/pull/3660
```